src/axis.ts

```diff
@@ -106,7 +106,7 @@ export interface Axis extends AxisOrientMixins, BaseAxisNoSignals, Guide {
   values?: number[] | string[] | boolean[] | DateTime[];
 
   /**
-   * A non-positive integer indicating z-index of the axis.
+   * A natural number indicating the z-index of the axis.
```

Comment:
Natural number doesn't have a consistent interpretation whether it includes zero (ISO standard includes 0 but layman's dictionary sometimes d o not), making it ambiguous.  

"non-negative integer" is clearer?

Reply:
Done.